### ISSUE TITLE
Why is development stopped?

### ISSUE THREAD

**Issue by burekas7** [elgatito/plugin.video.elementum#536]
Regarding to this message in version 0.1.45
:/
```
Development of *plugin.video.elementum* is stopped!
    - Do not expect any support, help and other things like that.
    - Source is open, so you can fork everything.
```

If someone is going to make a fork please keep us updated.
Thanks @elgatito 
We hope you will consider it again.

**Comment by zequi** [comment]
If it’s true that Elementum is going to die... it’s very sad !! 😔😔
I really love this addon. It’s the best one of all of the addons.

**Comment by elgatito** [comment]
> Why is development stopped?
@burekas7 that is a strange question, since this plugin is developed in a free time, without any money paid.

I'm leaving this issue opened, if anyone wants to introduce own fork - feel free to post it here.

**Comment by burekas7** [comment]
@elgatito 
I know you have done everything in your free time and I very appreaicate that, and I'm sure that a lot of people who using it think the same.
Just the timing is strange a little, and I don't have any complain, just curiosity.
I remeber the time that you have started this project as a fork of quasar and have taken it few steps further. So I hope someone proffessional as you will take it or maybe you will make a comeback ;)

**Comment by bam80** [comment]
@elgatito so what's happened? Just say it, we can all understand.
Cheers.

**Comment by elgatito** [comment]
@bam80 nothing happened. I just don't want to spend my time and money for this project.

**Comment by antonsoroko** [comment]
@elgatito 
Do you have any plans to write some development guide?
I understand that there was not development guide left in quasar repo so you had to learn how it works in a "hard way". But I guess "development guide" could give elementum second life.

Thank you again for your work and time.

**Comment by bam80** [comment]
@elgatito so you played it enough.. Are you going to pass maintainer-ship of this repo to someone else maybe?

**Comment by martinstz** [comment]
Thank you for all you have done!
You have contributed a lot to Kodi's development and thanks to you, Kodi's fans are so much more! Good luck in your endeavors! Honor and respect from Bulgaria!

**Comment by elgatito** [comment]
@bam80 @antonsoroko 
I can help with some questions regarding how things work. 
But surely cannot teach development. 
Writing development documentation takes a lot of time, and it can be that it is not needed by anyone.

**Comment by EcvAntonio** [comment]
A simple question, I think, I'm a week using elementum and it was working normally, to this day I just don't want to recognize torrent files, how do I solve it?

(use on androidtv ARM7 = arm)

(bug appeared after January update)

(on 64bit computer the elementum simply stops working and comes back, stopping streaming)

**Comment by luispabon** [comment]
I would recommend you downgrade to a previous version and disable updates until someone (if anybody) picks up elementum's development.

**Comment by luispabon** [comment]
@elgatito just to say thank you for all the hard work and I wish you the best on your future endeavors.

**Comment by burekas7** [comment]
@EcvAntonio 
Open a new issue for it, not here.
And read again what he said: Do not expect any support, help and other things like that
So until a new developer will take it, you will need to wait, unless @elgatito will give support from time to time.

**Comment by pavelmxx** [comment]
Прямо аккурат с окончанием поддержки, trakt перестал строить списки "в топе" и "в тренде". Также не дает переавторизовать trakt под предлогом "Unable to get trakt code: 403". Так понимаю, что кина не будет. Жаль, что не получится устроить какую-нибудь стабильную версию. Все постепенно развалится само собой без поддержки, по крайней мере, что касается api trakt, структуры торрент сайтов и прочих динамических вещей, насколько я понимаю. И к черту эту имплементацию новых фишек, лишь бы сил и желания у Вас, elgatito, на "support from time to time" по особым случаям хватило. В любом случае, спасибо за труды. За пару лет получил истинное удовольствие и удобство при использовании. На свою голову подсадил десятка два пользователей, которым теперь еще придется объяснять, почему кина не будет. Без преувеличения, это трагедия для kodi пользователей, расстройство только начавшегося года.

**Comment by elgatito** [comment]
> Прямо аккурат с окончанием поддержки, trakt перестал строить списки "в топе" и "в тренде". Также не дает переавторизовать trakt под предлогом "Unable to get trakt code: 403". Так понимаю, что кина не будет.

Это совпадение. В версии 0.1.47 Тракт снова работает и можно свой ключ прописать, при желании. Нужно пере-авторизовать  Тракт.

**Comment by pavelmxx** [comment]
> Это совпадение. В версии 0.1.47 Тракт снова работает и можно свой ключ прописать, при желании. Нужно пере-авторизовать Тракт.
Спасибо, помогло. У нас, оказывается, в стране surge.sh по ip заблочен теперь. Обновить удалось только вручную из скаченного zip.

**Comment by roliverosc** [comment]
Una gran pena que se pare el desarrollo de este maravilloso complemento. Saludos @elgatito

**Comment by splaquet** [comment]
I agree, it’s is sad. Elementum has also become my very favorite (possibly exclusively used?) add-on to date. I’d never heard of it, until Gaia Repo introduced it to me. 

On that note, the developer of Gaia has also said that he’s no longer developing on the current major version. Instead, he’s going to be focusing on his next major version release. 

FYI folks, I have a feeling that we’re going to see this a lot. Over the years, we’ve been fortunate enough to have players, such as @elgatito , jumping into projects and running with them. 

In short, having to redo everything from the ground up (ie: recoding everything for Python 3 compatibility) is pretty much starting from the beginning. 

Maybe finding someone willing to help convert the code to Python 3 would be the motivating factor that we need 😉

**Comment by RH1CK** [comment]
Its sad that you wont be supporting the addon, but I have to say thanks a thousand times, I used it a lot, I know how its to do something and not be paid for that

**Comment by hoshsadiq** [comment]
Another to add a thank you. It's been a great addon and will continue to use it until it fully breaks. Thank you for your support!

**Comment by BTopbas** [comment]
Another thank you too from me! I am using this addon since pulsar quasar. And this is like a flag race. Thanks for your fixes and new features!

**Comment by ETE-Design** [comment]
Anyone know what plugin to use now that support "Burst" plugin?

**Comment by borekon** [comment]
You can still use elementum.... @ETE-Design

**Comment by DejavuDmitry** [comment]
Народ, может как-то деньгами скинуться для основного разработчика? Едрить-колотить. Я готов! ＼(◎o◎)／

**Comment by antonsoroko** [comment]
@DejavuDmitry 
http://elementum.surge.sh/donate/

**Comment by giaur500** [comment]
You can try Kodi Popcorn Time. It's smaller and not so fat, as it uses Python libraries to download torrents:

https://github.com/markop159/KODI-Popcorn-Time/blob/master/README.md

It's also somehow maintained still

**Comment by borekon** [comment]
Main disadvantage of every other torrent platform:https://github.com/markop159/KODI-Popcorn-Time/wiki/FAQ#does-it-downloads-the-whole-file-do-i-need-the-space-is-it-ever-deletedOn Apr 26, 2020 23:40, giaur500 <notifications@github.com> wrote:
You can try Kodi Popcorn Time. It's smaller and not so fat, as it uses Python libraries to download torrents:
https://github.com/markop159/KODI-Popcorn-Time/blob/master/README.md
It's also somehow maintained still

**Comment by burekas7** [comment]
Guys, keep working with Elementum, everything is ok for now.
This is the best Kodi addon which have ever been made and it's still like this.

**Comment by antonsoroko** [comment]
@elgatito 
btw, maybe something like https://www.bountysource.com/teams/plugin.video.quasar will encourage new developers to contribute to the project? (e.g. users will post bugs and put a reward for fixing bugs)

Just a thought (I accidentally discovered this).

**Comment by giaur500** [comment]
All similar addons are dead completly, the same goes to kodi popcorn time.

**Comment by ra77767** [comment]
live alternative is [TorrServer ](https://github.com/YouROK/TorrServer/releases)

**Comment by burekas7** [comment]
> live alternative is [TorrServer ](https://github.com/YouROK/TorrServer/releases)

It doesn't seems to be something the relates for kodi.
Besides, Elementum is still working perfectly for now.

**Comment by kubo6472** [comment]
As of 2024 seems to be live enough. Any reason why the warning stayed?

**Comment by antonsoroko** [comment]
Maybe the message for element and burst should be changed to "Active development of this addon has been stopped.", instead of "Development of this addon has been stopped!", since some development is still going on.

**Comment by borekon** [comment]
> Maybe the message for element and burst should be changed to "Active development of this addon has been stopped.", instead of "Development of this addon has been stopped!", since some development is still going on. This change will make the message less confusing.

Bump! three years later.... lol

**Comment by elgatito** [comment]
I'd make some efforts to upgrade Elementum to newer environment, but also would be glad to get helpful code contributions.

**Comment by antonsoroko** [comment]
@elgatito what do you mean by "newer environment"? btw, when i had time i tried to (actually LLM tried, i just directed it) to update libtorrent to 2.0 (swig & c++ & go) and after many iterations it was successful but unstable. then my free time came to end.

**Comment by elgatito** [comment]
@antonsoroko Don't have a clear list yet.
